app/i18n/po/decrediton.pt.po

```diff
@@ -3323,13 +3323,13 @@ msgstr "Comprar Tickets"
 #. defaultMessage is:
 #. Purchase Tickets is not available right now,
 #.   because we are in the beginning of a ticket interval.
-#.   after {blocksNumber} blocks it will be available again.
+#.   After {blocksNumber} blocks it will be available again.
```

Comment:
No need to update the .po and .pot files.

These files are generated automatically based on the source strings before releases and sent for transactions.